aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -40,13 +40,13 @@ public class CanvasManager : MonoBehaviour
     /// Name of the start screen.
     /// </summary>
     [SerializeField]
-    private string _sceneNameStartingMenu = "Settings"; // TODO:: Load main start screen
+    private const string _sceneNameStartingMenu = "Settings"; // TODO:: Load main start screen
```

Comment:
I would make them readonly instead of const, now you cannot edit them in the editor anymore

```suggestion
    private readonly string _sceneNameStartingMenu = "Settings";
```

Reply:
Readonly doesnt work with switch cases. I converted all of them to if statements and removed the modifiers.